scripts/auth.js

```diff
@@ -0,0 +1,162 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let isEmailValid = false;
+  let isNicknameValid = false;
+  let isPasswordValid = false;
+  let isPasswordConfirmationValid = false;
+
+  const loginForm = document.querySelector('.login-form');
+  const signupForm = document.querySelector('.signup-form');
+  const emailInput = document.querySelector('#email');
+  const nicknameInput = document.querySelector('#nickname');
+  const passwordInput = document.querySelector('#password');
+  const passwordConfirmationInput = document.querySelector('#passwordConfirmation');
+  const submitButton = document.querySelector('button[type="submit"]');
+
+  // 오류 메세지 노출 함수
+  function showError(input, errorClass) {
+    const errorElement = document.getElementsByClassName(errorClass)[0];
+    errorElement.style.display = "block";
+    input.style.border = "1px solid #f74747";
+  }
+  
+  // 오류 메세지 숨김 함수
+  function hideError(input, errorClass) {
+    const errorElement = document.getElementsByClassName(errorClass)[0];
+    errorElement.style.display = "none";
+    input.style.border = "none";
+  }
+
+  // 이메일 형식 유효성 검사
+  function validateEmailString(email) {
```

Comment:
boolean 값이 리턴되는 함수의 경우에는 is~ 식의 네이밍을 추천드려요!
```suggestion
  function isEmailValid(email) {
```